XamlControlsGallery/ControlPages/SliderPage.xaml.cs

```diff
@@ -30,5 +30,63 @@ public SliderPage()
         {
             this.InitializeComponent();
         }
+
+
+        private void MinimumValue_ValueChanged(Microsoft.UI.Xaml.Controls.NumberBox sender, Microsoft.UI.Xaml.Controls.NumberBoxValueChangedEventArgs args)
+        {
+            //Handle exceptional scenario
+            if(args.NewValue >= Slider2.Maximum)
+            {
+                Slider2.Maximum = args.NewValue + 1;
+            }
+            Slider2.Minimum = args.NewValue;
+        }
+
+        private void MaximumValue_ValueChanged(Microsoft.UI.Xaml.Controls.NumberBox sender, Microsoft.UI.Xaml.Controls.NumberBoxValueChangedEventArgs args)
+        {
+            //Handle exceptional scenario
+            if(args.NewValue <= Slider2.Minimum)
+            {
+                Slider2.Minimum = args.NewValue - 1;
+                Slider2.Value = Slider2.Minimum;
+            }
+            Slider2.Maximum = args.NewValue;
+        }
+
+        private void StepFrequencyValue_ValueChanged(Microsoft.UI.Xaml.Controls.NumberBox sender, Microsoft.UI.Xaml.Controls.NumberBoxValueChangedEventArgs args)
+        {
+            try
+            {
+                Slider2.StepFrequency = (float)args.NewValue;
+            }
+            catch (Exception)
+            {
+                Slider2.StepFrequency = (Slider2.Maximum - Slider2.Minimum) / 100.0;
+            }
+        }
+
+        private void SmallChangeValue_ValueChanged(Microsoft.UI.Xaml.Controls.NumberBox sender, Microsoft.UI.Xaml.Controls.NumberBoxValueChangedEventArgs args)
+        {
+            try
+            {
+                Slider2.SmallChange = (float)args.NewValue;
+            }
```

Comment:
Same as above, can this fail or can we guard better against this or can we even use x:Bind?